src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/MessageProcessor.java

```diff
@@ -24,15 +24,20 @@
 
     @Scheduled(fixedDelayString = "${queue.read-interval}")
     public void run() {
+        logger.info("Started processing queue messages.");
+
         IMessage msg = null;
+        int processedMessagesCount = 0;
+
         try {
             IMessageReceiver msgReceiver = messageReceiverFactory.create();
             while ((msg = msgReceiver.receive()) != null) {
-
                 envelopeProcessor.onMessageAsync(msg).get();
-
                 msgReceiver.complete(msg.getLockToken());
+                processedMessagesCount++;
```

Comment:
I think we should be using a metric here and report this back ... especially how many errors.

https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-metrics.html

Reply:
I agree. It would be worth creating a story/task to add those.